MIP/mip-40/README.md

```diff
@@ -0,0 +1,83 @@
+
+We also propose the usage of intermediary tokens for bridging to fully decouple the L1 and L2 systems. This allows for the L2 pool to be recirculated without the need for a direct bridge to the L1 and exposure to fallibility of the bridge on either side. 
+
+Without the usage of intermediary tokens, we refer to this system as ***AB-FFS Partially Decoupled***, for which we do not intend a separate MIP. 
+
+## Motivation
+
+The *AB-FFS Decoupled* system is intended to address [MD-38](https://github.com/movementlabsxyz/MIP/pull/38). That is, *AB-FFS Decoupled* provides a means for using the Atomic Bridge and Fast-finality Settlement with a fixed supply of token. 
+
+Fundamentally, *AB-FFS Decoupled* contends with the fallibility of the Atomic Bridge by (a) removing any need for its involvement in returning funds to the L1 and (b) using intermediary tokens to avoid any locking, minting, burning, or other operations that would potentially modify the total supply of either the L1 staking token or the L2 staking token.
+
+*AB-FFS Partially Decoupled* still removes the need for the Atomic Bridge to return funds to the L1, but does not protect the total supply from all bridge failures. Hence *AB-FFS Partially Decoupled* is not a sound solution for [MD-38](https://github.com/movementlabsxyz/MIP/pull/38).
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+![*AB-FFS Decoupled*](./ab-ffs-decoupled.png)
+
+An *AB-FFS Decoupled* system is defined by the following requirements:
+
+1. The L1 staking token MUST be a unique token issued in a fixed supply.
+2. The L2 gas token MUST be a unique token issued in a fixed supply.
+3. Rewards issued via FFS MUST NOT be issued in the L1 staking token, but instead in an LP token on the L1.
+4. The LP Token MAY elect to maintain a supply or value correlated with the L1 staking token or the L2 gas token. But, this supply or value correlation MUST NOT rely on special capabilities of the LP token or trusted third parties.
+5. The L2 staking token MUST be burned on the L2 to deposit gas into an L2 pool.
+
+If the LP Token supply is not fixed, the AB-FFS solution implemented is referred to as *AB-FFS Decoupled* Demurrage. [MIP-n](todo) provides a more complete specification for *AB-FFS Decoupled* Demurrage. 
+
+Otherwise, the AB-FFS solution implemented implemented is referred to as *AB-FFS Decoupled* Grant. [MIP-n](todo) provides a more complete specification for *AB-FFS Decoupled* Grant.
+
+### L1 LP Token
+1. The L1 LP Token MUST NOT enshrine any special capabilities to mint or otherwise unlock the L1 staking token. That is, the L1 LP token must be independent of the L1 staking token and only effectively be used to purchase the L1 staking token.
```

Comment:
The idea is to preclude any kind of capabilities which would allow minting or unlocking token, at all, as any form of side-effect.

Reply:
```suggestion
1. The L1 LP Token MUST be designed such that it cannot be directly used to mint or otherwise unlock the L1 staking token. That is, the L1 LP token must be independent of the L1 staking token and only effectively be used to purchase the L1 staking token.
```

how about this?